upload/lambdas/api_server/v1/area.py

```diff
@@ -56,7 +57,7 @@ def store_file(upload_area_uuid: str, filename: str, body: str):
 @return_exceptions_as_http_errors
 def file_uploaded_notification(upload_area_uuid: str, filename: str):
     upload_area = _load_upload_area(upload_area_uuid)
-    upload_area.add_uploaded_file_to_csum_daemon_sqs(filename)
+    CsumSQSQueue(upload_area, filename).enqueue()
```

Comment:
It seems that enqueue would take filename as a param rather than instantiating CsumSQSQueue with filename. This is related to another comment